src/engine/enginetalkoverducking.cpp

```diff
@@ -64,16 +64,11 @@ void EngineTalkoverDucking::slotDuckModeChanged(double mode) {
    m_pConfig->set(ConfigKey(m_group, "duckMode"), ConfigValue(mode));
 }
 
-CSAMPLE EngineTalkoverDucking::getGain(int numFrames) {
+double EngineTalkoverDucking::getGain() {
     // Apply microphone ducking.
-    switch (getMode()) {
-      case EngineTalkoverDucking::OFF:
+    if (getMode() == EngineTalkoverDucking::OFF) {
         return 1.0;
-      case EngineTalkoverDucking::AUTO:
-        return calculateCompressedGain(numFrames);
-      case EngineTalkoverDucking::MANUAL:
+    } else {
```

Comment:
... or should I use the switch like before, simply return m_pDuckStrength->get() for Auto and Manual and keep the warning in place?

Reply:
This change means that auto is no longer auto, it's just using the manual setting.  The point of auto is that it's supposed to pick a good level to compress the audio to.  If it's picking bad numbers, then we need to fix calculateCompressedGain